Subject: Remove fpu_control.h include
Upstream: No
Author: Simon Frankenberger <simon-alpine@fraho.eu>

The header is not present with musl and including it results in build error.
It's not needed anyways.

The second patch fixes "error: redeclaration of 'using MacroAssembler::call_VM_leaf_base'"

--- old/src/hotspot/os_cpu/linux_aarch64/os_linux_aarch64.cpp
+++ new/src/hotspot/os_cpu/linux_aarch64/os_linux_aarch64.cpp
@@ -77,7 +77,6 @@
 # include <pwd.h>
 # include <poll.h>
 # include <ucontext.h>
-# include <fpu_control.h>
 
 #ifdef BUILTIN_SIM
 #define REG_SP REG_RSP
--- old/src/hotspot/cpu/aarch64/interp_masm_aarch64.hpp
+++ new/src/hotspot/cpu/aarch64/interp_masm_aarch64.hpp
@@ -38,8 +38,6 @@
  protected:
 
  protected:
-  using MacroAssembler::call_VM_leaf_base;
-
   // Interpreter specific version of call_VM_base
   using MacroAssembler::call_VM_leaf_base;
 
